### PR TITLE
Add orbit information to node running page

### PR DESCRIPTION
**Pull request by Jason-W123**

### CODE REVIEW

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 1]
```diff
@@ -37,14 +37,21 @@ The minimum storage requirements will change over time as the Nitro chain grows.
 
 ### Required parameters
 
-- L1 RPC URL
-  - Use the parameter `--parent-chain.connection.url=<Layer 1 Ethereum RPC URL>`
-  - It must provide a standard layer 1 node RPC endpoint that you run yourself or from a node provider
+- Parent chain RPC URL
+  - Use the parameter `--parent-chain.connection.url=<Parent chain RPC URL>`
+  - It must provide a standard evm node RPC endpoint that you run yourself or from a node provider
```
Comment (mahsamoosavi) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:42]:
```suggestion
  - It must provide a standard RPC endpoint for an EVM node, whether self-hosted or obtained from a node service provider
```

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 2]
```diff
@@ -37,14 +37,21 @@ The minimum storage requirements will change over time as the Nitro chain grows.
...
+  - Would be set to appropriate Arb1, Nova, Sepolia RPC URL
```
Comment (mahsamoosavi) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:43]:
```suggestion
  - Would be set to appropriate Arbitrum One, Arbitrum Nova, or Arbitrum Sepolia RPC URL
```

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 3]
```diff
@@ -37,14 +37,21 @@ The minimum storage requirements will change over time as the Nitro chain grows.
...
+  - Note: [Arbitrum.io public RPCs](/node-running/node-providers.mdx) will rate limit connections, will need to run local node for parent chain or use 3rd party RPC providers
```
Comment (mahsamoosavi) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:44]:
```suggestion
  - Note: [Arbitrum.io public RPCs](/node-running/node-providers.mdx) will rate limit connections, you will need to run a local node for the parent chain or relay on third party RPC providers
```

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 4]
```diff
@@ -37,14 +37,21 @@ The minimum storage requirements will change over time as the Nitro chain grows.
...
   - Note: this parameter was called --l1.url in versions prior to `v2.1.0`
 - L2 chain id or name
   - Use the parameter `--chain.id=<L2 chain ID>` to set the L2 chain from its chain id. See [RPC endpoints and providers](/node-running/node-providers.mdx#rpc-endpoints) for a list of Arbitrum chains and their respective L2 chain ids.
   - Alternatively, you can use the parameter `--chain.name=<L2 chain name>` to set the L2 chain from its name (options are: `arb1`, `nova`, `goerli-rollup` and `sepolia-rollup`)
   - Note: this parameter was called --l2.chain-id and only accepted chain ids in versions prior to `v2.1.0`
+- For Orbit chains
+  - `nodeConfig.json` will be generated when orbit chain is created, you need to set it using either `--conf.file` or `--chain.info-json`
```
Comment (mahsamoosavi) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:51]:
```suggestion
  - `nodeConfig.json` will be generated when an Orbit chain is created, you need to set it using either `--conf.file` or `--chain.info-json` 
```

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 5]
```diff
@@ -37,14 +37,21 @@ The minimum storage requirements will change over time as the Nitro chain grows.
...
+  - Would be set to appropriate Arb1, Nova, Sepolia RPC URL
+  - Note: [Arbitrum.io public RPCs](/node-running/node-providers.mdx) will rate limit connections, will need to run local node for parent chain or use 3rd party RPC providers
   - Note: this parameter was called --l1.url in versions prior to `v2.1.0`
 - L2 chain id or name
   - Use the parameter `--chain.id=<L2 chain ID>` to set the L2 chain from its chain id. See [RPC endpoints and providers](/node-running/node-providers.mdx#rpc-endpoints) for a list of Arbitrum chains and their respective L2 chain ids.
   - Alternatively, you can use the parameter `--chain.name=<L2 chain name>` to set the L2 chain from its name (options are: `arb1`, `nova`, `goerli-rollup` and `sepolia-rollup`)
   - Note: this parameter was called --l2.chain-id and only accepted chain ids in versions prior to `v2.1.0`
+- For Orbit chains
+  - `nodeConfig.json` will be generated when orbit chain is created, you need to set it using either `--conf.file` or `--chain.info-json` 
+  - `--conf.file` is the docker container's path of `nodeConfig.json`, you need put this file to mount point first. `--chain.info-json` is the json content itself
```
Comment (mahsamoosavi) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:52]:
```suggestion
  - `--conf.file` refers to the Docker container's location of `nodeConfig.json`. Initially, you must place this file at the designated mount point. As for -`-chain.info-json`, it represents the actual JSON content
```

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 6]
```diff
@@ -37,14 +37,21 @@ The minimum storage requirements will change over time as the Nitro chain grows.
...
+  - `--chain.name="My Arbitrum L3 Chain"` must be provided, and matches the chain name used in `--conf.file` file or `--chain.info-json` content
```
Comment (mahsamoosavi) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:53]:
```suggestion
  - `--chain.name="My Arbitrum L3 Chain"` must be provided, and it must match the chain name that is used in `--conf.file` file or `--chain.info-json` content
```

File: arbitrum-docs/node-running/how-tos/running-a-full-node.mdx [hunk 7]
```diff
@@ -37,14 +37,21 @@ The minimum storage requirements will change over time as the Nitro chain grows.
...
+  - If anytrust chain, also need to supply `--node.data-availablity.enable` and then some ways to get the data like `--node.data-availability.rest-aggregator.enable` and associated options or `--node.data-availability.ipfs-storage.enable` and associated options
```
Comment (mahsamoosavi) [inline arbitrum-docs/node-running/how-tos/running-a-full-node.mdx:54]:
```suggestion
  - For an Anytrust chain, it is essential to provide `--node.data-availability.enable`. Additionally, you must choose a method for obtaining the data, such as enabling `--node.data-availability.rest-aggregator.enable` with its associated options, or opting for `--node.data-availability.ipfs-storage.enable` along with its associated options
```

**Review by mahsamoosavi — CHANGES_REQUESTED** [review]

**Comment by Jason-W123** [comment]
Close it as we have new plan

**Pull request closed without merging**